src/config.js

```diff
@@ -0,0 +1,91 @@
+export let colors = {
+  primaries: ['#0076de', '#0061b6', '#004c8e', '#b9d9f6'],
+  secondaries: ['#3a3f52', '#272c41', '#202536'],
+  neutrals: [
+    '#ffffff',
+    '#f9f9f9',
+    '#f4f4f4',
+    '#ededed',
+    '#e8e8e8',
+    '#d3d3d3',
+    '#aaaaaa',
+    '#777777',
+    '#454545',
+  ],
+  errors: ['#ff5630', '#de350b', '#bf2600', '#ffbdad'],
+  successes: ['#5bb85b', '#4b974b', '#3a763a', '#a5d8a5'],
+  infos: ['#6554c0', '#5243aa', '#403294', '#c0b6f2'],
+  warning: '#ffab00'
+}
+
+export let fontFamilies = { primary: 'Lato' }
+
+let common = {
```

Comment:
This might be a bit too generically named given that it’s only the common font properties between text components